src/walberla_bridge/src/lattice_boltzmann/LBWalberlaImpl.hpp

```diff
@@ -40,7 +40,8 @@
 #include <stencil/D3Q27.h>
 #if defined(__CUDACC__)
 #include <gpu/AddGPUFieldToStorage.h>
-#include <gpu/communication/GPUPackInfo.h>
+#include <gpu/communication/MemcpyPackInfo.h>
+#include <gpu/communication/UniformGPUScheme.h>
```

Comment:
For more details about packing and buffered schemes with GPU fields, please refer to [walberla/walberla!672](https://i10git.cs.fau.de/walberla/walberla/-/merge_requests/672).